src/Installer/ApigeeDevportalKickstartTasksManager.php

```diff
@@ -97,6 +98,27 @@ public static function createPaymentGateway(array $values, array &$context) {
       $context['results']['$gateway'] = $gateway;
       $context['message'] = t('Created a default payment gateway.');
     }
+    catch (\Exception $exception) {
```

Comment:
See message above, `createPaymentGateway` is not being called, but should be

Reply:
Fixed :)